doc/man3/OSSL_CMP_sk_X509_add1_cert.pod

```diff
@@ -0,0 +1,62 @@
+=pod
+
+=head1 NAME
+
+OSSL_CMP_sk_X509_add1_cert,
+OSSL_CMP_sk_X509_add1_certs,
+OSSL_CMP_X509_STORE_add1_certs,
+OSSL_CMP_X509_STORE_get1_certs,
+OSSL_CMP_build_cert_chain
+- functions manipulating lists of certificates
+
+=head1 SYNOPSIS
+
+  #include <openssl/cmp_util.h>
+
+  int OSSL_CMP_sk_X509_add1_cert(STACK_OF(X509) *sk, X509 *cert,
+                                 int not_duplicate, int prepend);
+  int OSSL_CMP_sk_X509_add1_certs(STACK_OF(X509) *sk,
+                                  const STACK_OF(X509) *certs,
+                                  int no_self_signed, int no_duplicates);
+  int OSSL_CMP_X509_STORE_add1_certs(X509_STORE *store, STACK_OF(X509) *certs,
+                                int only_self_signed);
+  STACK_OF(X509) *OSSL_CMP_X509_STORE_get1_certs(X509_STORE *store);
+  STACK_OF(X509) *OSSL_CMP_build_cert_chain(STACK_OF(X509) *certs, X509 *cert);
```

Comment:
None of these feel CMP specific. Either they should not be part of the public API, or they should not be in CMP.

Reply:
As answered above, we must publish these utility functions for upcoming use in the CMP app,
and I'm open to move them wherever they fit better.

Reply:
As mentioned above, I've made them internal.